server/hedley/modules/custom/hedley_admin/hedley_admin.module

```diff
@@ -308,3 +317,95 @@ function hedley_admin_health_center_access($account = FALSE) {
     return FALSE;
   }
 }
+
+/**
+ * Implements hook_form() for the suspected devices state table.
+ *
+ * This function generates a table displaying information
+ * for devices that are suspected to have a synchronization problem.
+ */
+function hedley_admin_devices_sync_state_table(array $form, array &$form_state) {
+  // Table header definitions.
+  $header = array(
+    'index' => t('#'),
+    'name' => t('Device Name'),
+    'to_upload' => [
+      'data' => t('Changes to Upload'),
+      'class' => 'align-center',
+    ],
+    'last_sync' => [
+      'data' => t('Last Sync Contact'),
+      'class' => 'align-center',
+    ],
+  );
+
+  // SQL query to retrieve synchronization data.
+  $query = db_select('node', 'n');
+  $query->join('users', 'u', 'u.uid = n.uid');
+  $query->join('field_data_field_total_to_upload', 'ttu', 'ttu.entity_id = u.uid');
+  $query->join('field_data_field_sync_phase', 'sp', 'sp.entity_id = u.uid');
+  $query->addField('u', 'name');
+  $query->addField('ttu', 'field_total_to_upload_value');
+  $query->addExpression('FROM_UNIXTIME(u.access)', 'last_sync');
+  $query->condition('n.type', 'sync_incident');
+  $query->condition('sp.field_sync_phase_value', 'sync-start');
+  $query->condition('ttu.field_total_to_upload_value', 10, '>');
+  $query->condition('u.access', strtotime('-2 months'), '>');
+  $query->groupBy('u.name');
+  $query->orderBy('last_sync', 'DESC');
+
+  $result = $query->execute();
+
+  $rows = [];
+  foreach ($result as $key => $row) {
+    $words = explode(' ', $row->name);
+    if (end($words) == 'Robot') {
+      array_splice($words, -1);
```

Comment:
I am not sure why does it make sense to eliminate the "Robot" word, if it's for admin and super-admin, but surely it makes no harm.

Reply:
When device is paired, user is created for the device.
When created, user title is set to '$device_title Robot", to indicate that user was created automatically for the device.
So, "Robot" word is dropped to present the name of actual device.